src/app/content/components/popUp/ChapterFilter.tsx

```diff
@@ -23,6 +28,7 @@ const Row = styled.div`
 const Column = styled.div`
   display: flex;
   flex-direction: column;
+  overflow-x: hidden;
```

Comment:
I couldn't find a way to get rid out of this:
![image](https://user-images.githubusercontent.com/31478212/99279565-a6fbe780-2830-11eb-98ab-485ec8989c5f.png)

Reply:
are you talking about the orange bars? what is that horizontal line at the top of the dropdown it looks like its a few px out of place

Reply:
about the bottom one
I couldn't find why it appears

> it looks like its a few px out of place

I think this is because of how Firefox displays the scroll bars - when you hover on them they are getting wider and fill the rest of the space

Reply:
the second thing i said was asking about the grey line at the top of the dropdown, it should line up with the top right? it looks like its 2px down